statistics/statistics.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Rubik&display=swap" rel="stylesheet" />
+    <link href="statistics.css" rel="stylesheet" />
+    <title>Statistics page</title>
+</head>
+<body>
+    <header class="header">
+        <fiqure class="header-img">
+            <img src="/images/corona3.png"  alt="Header corona image">
+        </fiqure>
+        <section class="header-locatin ">
+            <img src="/images/loction.png" width="20px"  alt="">
+            <h1 class="country">
+                Palestine
+            </h1> 
+        </section>
+        <section>
```

Comment:
use `<figure>` for all your `<img>`s not `<section>`
`<section>` can be used as a bigger container and that's depends to your need
like this
```
<header></header>
<section>
	<article>
		<figure>
			<img>
			<figcaption></figcaption>
		</figure>
	</article>
</section>
<footer></footer>
```